.github/workflows/make-release.yml

```diff
@@ -0,0 +1,120 @@
+name: 'PyCyphal application builder'
+
+# Builds OpenCyphal PyCyphal release
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+# Ensures that only one workflow is running at a time
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  #
+  # PyCyphal test
+  #
+  pycyphal-test:
+    name: Test PyCyphal
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-2019-npcap]
```

Comment:
```suggestion
        os: [ubuntu-latest, windows-2019-npcap]
```

Reply:
The tests will now run on `ubuntu-latest` runners as requested.